jcl/src/java.management/share/classes/module-info.java.extra

```diff
@@ -22,10 +22,9 @@
  *******************************************************************************/
 module java.management {
 	requires java.logging;
-	exports com.ibm.java.lang.management.internal to com.ibm.management, jdk.management;
-	exports sun.management.spi to com.ibm.management;
```

Comment:
is `sun.management.spi` to be exported to `jdk.management` now?

Reply:
It is already exported to `jdk.management` by the original `java.management` module.